storage/timecache/timeCacher.go

```diff
@@ -81,7 +81,7 @@ func (tc *timeCacher) Clear() {
 
 // Put adds a value to the cache. It will always return false since the eviction did not occur
 func (tc *timeCacher) Put(key []byte, value interface{}, _ int) (evicted bool) {
-	_, err := tc.timeCache.upsert(string(key), value, tc.timeCache.defaultSpan)
+	_, err := tc.timeCache.put(string(key), value, tc.timeCache.defaultSpan)
```

Comment:
So all callers of `timeCacher.Put()` will receive this fix. Should be only positive impact, correct?

Reply:
right, this was the actual fix, a complete rewrite of the existing data